docs/concepts.md

```diff
@@ -2,19 +2,19 @@
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 **Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
 
-- [Kubefed Concepts](#kubefed-concepts)
+- [KubeFed Concepts](#kubefed-concepts)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-### Kubefed Concepts
+### KubeFed Concepts
 
 | Concept              | Description                                                                                                                                                                                                                                                                                                         |
 | -------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | Federate             | Federating a set of Kubernetes clusters means, effectively creating a common interface to the pool of these clusters which can be used to deploy Kubernetes applications across those clusters.                                                                                                                     |
-| Federation           | Kubernetes Cluster Federation enables users to federate multiple Kubernetes clusters for resources distribution, service discovery, high availability etc across multiple clusters.                                                                                                                                 |
-| Host Cluster         | A cluster which is used to expose the federation API and run the kubefed control plane.                                                                                                                                                                                                                          |
+| KubeFed           | KubeFed enables users to federate multiple Kubernetes clusters for resources distribution, service discovery, high availability etc across multiple clusters.                                                                                                                                 |
```

Comment:
Please revert replacement in second column

Reply:
Done